src/Command/UpdatePullRequestStatusCommand.php

```diff
@@ -73,7 +73,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             'context'     => 'tower/pr-builder',
         ];
 
-        $response = $this->client->api('repos')->statuses()->create($this->githubOwner, $this->githubRepository, $sha1, $params);
-        $output->writeln($response['id']);
+        try {
+            $response = $this->client->api('repos')->statuses()->create($this->githubOwner, $this->githubRepository, $sha1, $params);
+            $output->writeln($response['id']);
+
+            return 0;
+        } catch (\Exception $e) {
+            $output->writeln($e->getMessage());
```

Comment:
Préférer utiliser `$this->getApplication()->renderThrowable()` (pour mieux mettre en forme l'exception) + utilise un logger